packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.test.ts

```diff
@@ -143,13 +143,20 @@ const tests = (Object.keys(mutationPayloads) as ShopperBasketsMutationType[]).ma
                             }
                         })
 
-                        await waitForValueToChange(() => result.current.queries.basket.data)
+                        const initialData = result.current.queries.basket.data
+                        await waitFor(() => {
+                            expect(result.current.queries.basket.data).not.toBe(initialData)
+                        })
 
                         act(() => {
                             result.current.mutation.mutate(payload)
                         })
 
-                        await waitForValueToChange(() => result.current.mutation.isSuccess)
+                        const initialSuccess = result.current.mutation.isSuccess
+                        await waitFor(() => {
+                            expect(result.current.mutation.isSuccess).not.toBe(initialSuccess)
+                        })
+
                         expect(result.current.mutation.isSuccess).toBe(true)
 
                         // On successful mutation, the query cache gets updated too. Let's assert it.
```

Comment:
The assertion for `invalidate?.forEach` and `remove?.forEach` fail. To be honest I don't really know why. 🤷‍♂️